app/modules/annotations/models.py

```diff
@@ -147,6 +147,10 @@ def set_bounds(self, bounds):
         self.validate_bounds(bounds)
         self.bounds = bounds
 
+    def get_image_url(self):
+        # TODO this needs a new Jira
```

Comment:
Or does it, ongoing discussion at the moment, this is the primary reason this is still draft.